src/app/core/components/login/login.component.ts

```diff
@@ -13,11 +14,19 @@ export class LoginComponent implements OnInit {
     password: ''
   };
 
-  constructor(private auth: AuthService) {}
+  constructor(private auth: AuthService, private router: Router) { }
 
-  ngOnInit() {}
+  ngOnInit() { }
 
   googleLogin() {
     this.auth.googleLogin().catch(error => (this.loginError = error));
   }
+
+  emailLogin() {
+    this.auth
+      .emailLogin(this.user.email, this.user.password)
+      .catch(error => (this.loginError = error))
+      .then(() => this.router.navigate(['']));
```

Comment:
Change the router navigation to go to the user's dashboard

Reply:
Fixed